pipeline/secret.go

```diff
@@ -105,26 +105,21 @@ func (s *Secret) ParseOrg(org string) (string, string, error) {
 	// check if the secret is not a native or vault type
 	if !strings.EqualFold(s.Engine, constants.DriverNative) &&
 		!strings.EqualFold(s.Engine, constants.DriverVault) {
-		return "", "", fmt.Errorf("%s: %s", ErrInvalidEngine, s.Engine)
-	}
-
-	// check if a path was provided
-	if !strings.Contains(path, "/") {
-		return "", "", fmt.Errorf("%s: %s ", ErrInvalidPath, path)
+		return "", "", fmt.Errorf("%w: %s", ErrInvalidEngine, s.Engine)
 	}
 
 	// split the full path into parts
-	parts := strings.SplitN(path, "/", 2)
+	parts := strings.Split(path, "/")
```

Comment:
I'm not sure this will work, because you can have secrets with a name like `docker/username` so an org-level secret's key could be `<org>/docker/username`. At least that's a possibility today.

if you add a test case for that you will see that this change will break those type of secrets.

Reply:
Ahah, I hadn't realized there was a possibility to have a sort of "nested" secret at the org level. Need more tests in place to demonstrate that behavior :-P

Reply:
@wass3r Is it true to say that secret names can contain any arbitrary length of forward slashes in it? Is there a restricted set of characters somewhere for the naming?

Reply:
yea, no limitation that i'm aware of.